test/unit/strategies/BaseStrategyUnit.tree

```diff
@@ -0,0 +1,55 @@
+BaseStrategy::__BaseStrategy_init
+├── It should call onlyAllo
+├── When poolId is zero
+│   └── It should revert
+├── When poolId argument is zero
+│   └── It should revert
+└── It should set poolId
+
+BaseStrategy::increasePoolAmount
+├── It should call onlyAllo
+├── It should call _beforeIncreasePoolAmount
+├── It should add amount to poolAmount
+└── It should call _afterIncreasePoolAmount
+
+BaseStrategy::withdraw
+├── When parameters are valid
+│   ├── It should call onlyPoolManager
+│   ├── It should call _beforeWithdraw
+│   ├── It should call balanceOf at token
+│   ├── It should call transfer at token
+│   ├── It should call _afterWithdraw
```

Comment:
lets also add maybe, `It should not change the pool amount`

Reply:
Sure, will add it after #17 is merged